app/views/foods/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<div>
+<% @foods.each do |food| %>
```

Comment:
[] kindly add bootstrap styling to this loop

Reply:
Yeah I never started the styling, I only requested because you asked...